R/bambu_utilityFunctions.R

```diff
@@ -86,24 +88,22 @@ checkInputs <- function(annotations, reads, readClass.file,
                         Please re-create your annotation object')
             }
         } else {
-            stop("The annotations is not a GRangesList object.")
+            stop("The annotations is not a GRangesList object a TxDb or a path to a .gtf.")
         }
     } else {
         stop("Annotations is missing.")
     }
-    ## When SE object from bambu.quantISORE is provided ##
-    if (!is.null(reads) & (!is.null(readClass.file))) stop("At least bam file or
-        path to readClass file needs to be provided.")
```

Comment:
do you want to keep a warning for the case when no reads are provided? Just to catch this very basic user error?

Reply:
Reads isn't an optional argument anymore, R by default will give an error if nothing is provided